crates/context/config/src/client/protocol/evm.rs

```diff
@@ -89,14 +98,15 @@ impl<'a> EvmTransport<'a> {
         let mut networks = BTreeMap::new();
 
         for (network_id, network_config) in &config.networks {
-            let _ignored = networks.insert(
-                network_id.clone(),
-                Network {
-                    client: ClientBuilder::default().http(network_config.rpc_url.clone()),
-                    rpc_url: network_config.rpc_url.clone().to_string(),
-                    secret_key: network_config.access_key.clone(),
-                },
-            );
+            let signer: PrivateKeySigner =
+                PrivateKeySigner::from_str(&network_config.access_key).unwrap();
+            let wallet = EthereumWallet::from(signer);
+
+            let provider = ProviderBuilder::new()
```

Comment:
```suggestion
            let provider = Provider::builder()
```

Reply:
Was throwing issue so didn't update

Reply:
what was the error? you probably needed to specify `N` in `Provider<N>`

```rust
Provider::<Ethereum>::builder()
```

but that's me guessing